runtime/src/chpl-visual-debug.c

```diff
@@ -276,17 +286,18 @@ void chpl_vdebug_pause (int tagno) {
 //                  typeIndex length lineNumber fileName
 //
 
-void chpl_vdebug_log_put_nb (void *addr, c_nodeid_t node, void* raddr,
-                             size_t size, int32_t typeIndex,
-                             int ln, int32_t fname) {
+void cb_comm_put_nb (const chpl_comm_cb_info_t *info) {
   if (chpl_vdebug) {
     struct timeval tv;
+    const struct chpl_comm_info_comm *cm = &info->iu.comm;
     chpl_taskID_t commTask = chpl_task_getId();
     (void) gettimeofday (&tv, NULL);
-    chpl_dprintf (chpl_vdebug_fd, "nb_put: %lld.%06ld %d %d %lu 0x%lx 0x%lx %d %d %d %d %d\n",
-                  (long long) tv.tv_sec, (long) tv.tv_usec,  chpl_nodeID, node,
-                  (unsigned long) commTask, (long) addr, (long) raddr, 1, typeIndex, (int)size,
-                  ln, fname);
+    chpl_dprintf (chpl_vdebug_fd, 
+                  "nb_put: %lld.%06ld %d %d %lu 0x%lx 0x%lx %d %d %d %d %d\n",
```

Comment:
The pointer-typed cm->addr and cm_raddr values can be printed directly using the %p conversion specifier.  It would be a little clearer, but maybe there's a reason this isn't desirable, for example you need to scanf() them back in and want to manipulate them as long ints at that time?